packages/configuration-accessibility/README.md

```diff
@@ -9,13 +9,11 @@ This package will automatically install all the missing dependencies.
 To use it you will have to install it via `npm`:
```

Comment:
Do we still need to call this out since you get this package automatically with any install of `hint`? Same goes for any of our packages that get pulled as part of either `configuration-web-recommended` or `configuration-development`.

Reply:
This is for what's showed in npm. Right now those packages get pulled automatically via hint but that could change in the future.

Maybe I could add a note in the hints that are installed automatically via web-recommended and a11y? Same for those configurations.